TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/chassis/ChassisCommands.java

```diff
@@ -0,0 +1,26 @@
+package org.firstinspires.ftc.teamcode.subsystems.chassis;
+
+import com.arcrobotics.ftclib.command.Command;
+import com.arcrobotics.ftclib.command.InstantCommand;
+import com.arcrobotics.ftclib.command.RunCommand;
+import com.arcrobotics.ftclib.gamepad.GamepadEx;
+import com.qualcomm.robotcore.hardware.Gamepad;
+
+public class ChassisCommands {
+
+    public static Command getMoveByGamepad(GamepadEx usedGamepad) {
```

Comment:
think about getting suppliers of the velocities instead of game pad, so you don't force the user to use left x left y right x